colossalai/context/moe_context.py

```diff
@@ -21,6 +21,7 @@ def __init__(self):
         self.max_ep_size = None
         self.min_dp_size = None
```

Comment:
This class does not have `__enter__` or `__exit__` method. It seems that it's not a context manager. By the way, why is it asingleton?

Reply:
it is just a gloabl value to store some settings and is designed by haichen

Reply:
Can you move it to `colossalai/moe`? And `colossalai/nn/layer/moe` can move there too.

Reply:
ok. i will do it in the following pr